provision-osx/tasks/install_ruby.yml

```diff
@@ -0,0 +1,53 @@
+---
+
+-
+  name: Copy prerequisite gpg key
+  get_url:
+    url: "{{rvm_gpg_url}}"
+    dest: "{{remote_tmp_path}}/{{rvm_gpg_file_name}}"
+  environment:
+    http_proxy: "{{ proxy_url | default('') }}"
+    https_proxy: "{{ proxy_url | default('') }}"
+
+-
+  name: Install rvm prerequisites
+  shell: "/usr/local/bin/gpg --import {{remote_tmp_path}}/{{rvm_gpg_file_name}}"
+  args:
+    executable: /bin/bash
+  register: list
+
+-
+  name: Retrieve rvm install script
+  get_url: 
+    url: "{{rvm_install_url}}"
+    dest: "{{remote_tmp_path}}/{{rvm_install_file_name}}"
+  register: rvm_prerequisites
+  environment:
+    http_proxy: "{{ proxy_url | default('') }}"
+    https_proxy: "{{ proxy_url | default('') }}"
+
+-
+  name: Install rvm
+  shell: "bash {{remote_tmp_path}}/{{rvm_install_file_name}} stable"
+
+- 
+  name: Install ruby
+  shell: "source ~/.bash_profile && rvm install {{ ruby_version }}"
+  args:
+    executable: /bin/bash
+
+-
+  name: Use ruby version
+  shell: "source ~/.bash_profile && rvm alias create default {{ ruby_version }}"
+  args:
+    executable: /bin/bash
+
+- 
+  name: Install gem packages
+  shell: "source ~/.bash_profile && gem install --no-ri --no-rdoc {{item.name}} -v {{ item.version | default('\"\"') }}"
```

Comment:
I can't remember why I removed using the gem module. Will reimplement it and see, should work fine though.